src/plugins/intel_cpu/src/emitters/plugin/x64/jit_bf16_emitters.hpp

```diff
@@ -33,45 +33,63 @@ class jit_uni_vcvtneps2bf16 : public jit_emitter {
         }
     }
 
+    template <typename Vmm>
+    void saturate_input(Vmm& clamped,
+                        const Vmm& in,
+                        const std::string& bf16_min_key,
+                        const std::string& bf16_max_key) const {
+        Vmm vmm_temp = Vmm(aux_vec_idxs[1]);
+        h->uni_vmovups(vmm_temp, table_val(bf16_min_key));
+        h->uni_vmaxps(clamped, in, vmm_temp);
+        h->uni_vmovups(vmm_temp, table_val(bf16_max_key));
+        h->uni_vminps(clamped, clamped, vmm_temp);
+
+        if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core)) {
+            h->uni_vmovups(vmm_temp, table_val("selector"));
+            h->vfixupimmps(clamped, in, vmm_temp, 0);
+        }
+    }
+
     template <dnnl::impl::cpu::x64::cpu_isa_t isa>
     void emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const {
         using namespace Xbyak;
         using Vmm = typename dnnl::impl::utils::conditional3<isa == dnnl::impl::cpu::x64::sse41, Xmm, isa == dnnl::impl::cpu::x64::avx2, Ymm, Zmm>::type;
 
         Vmm in = Vmm(in_vec_idxs[0]);
+        Vmm clamped = Vmm(aux_vec_idxs[0]);
+        saturate_input(clamped, in, "bf16_min", "bf16_max");
```

Comment:
We can consider reuse Vmm "in" with Vmm "clamped". Vmm "in" will store clamped data and save one aux. clamp does not break "keep_source_intact" as I understand.

Reply:
`saturate_input(in, in, "bf16_min", "bf16_max");` will make the wrong outputs, maybe because the temp data will need this original 'in' data?

Reply:
My point is:
```
saturate_input(clamped, in, "bf16_min", "bf16_max");
h->uni_vmovups(in, clamped);
```
clamped is just a aux vec, do not hold for long. Just a taste.